src/components/AlbumName.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types'
+
+// Album Name
+export const AlbumName = ({ name, albumUrl }) => {
```

Comment:
Good practise to only pass along the data that this component needs to know about.